verbit/streaming_client.py

```diff
@@ -54,16 +56,20 @@ def __init__(self, access_token, on_media_error: typing.Callable[[Exception], No
         self._model_id = None
         self._language_code = None
 
-        # websocket
+        # server url
         self._schema = 'wss'
         self._base_url = "speech.verbit.co"
         self._server_path = '/ws'
+
+        # websocket
         self._ws_client = WebSocket(enable_multithread=True)
```

Comment:
^ I suggest setting this to `None` here, and create the `WebSocket()` instance on `.start_stream()`, the unittest Mocks have to be slightly changed for that... I'll try doing it.
Alternative: Make all these runtime-state in a contained class, that is re-created on start (and re-start)